sdk/synapse/synapse-spark/test/utils/recorderUtils.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { isPlaybackMode } from "@azure/test-utils-recorder";
+import { isNode } from "@azure/core-http";
+import * as dotenv from "dotenv";
+
+if (isNode) {
+  dotenv.config();
+}
+
+export function uniqueString(): string {
+  return isPlaybackMode()
+    ? ""
+    : Math.random()
```

Comment:
any reason not to use a guid here?

Reply:
Is it needed? I refer it  from  [https://github.com/Azure/azure-sdk-for-js/blob/1f8a171a648dcd53de6ebd4b8e5d420fa9c54abf/sdk/keyvault/keyvault-secrets/test/utils/recorderUtils.ts#L12](https://github.com/Azure/azure-sdk-for-js/blob/1f8a171a648dcd53de6ebd4b8e5d420fa9c54abf/sdk/keyvault/keyvault-secrets/test/utils/recorderUtils.ts#L12)

---
In reply to: [521615219](https://github.com/Azure/azure-sdk-for-js/pull/12487#discussion_r521615219) [](ancestors = 521615219)

Reply:
@sadasant any thoughts?